packages/google_maps_flutter/google_maps_flutter/ios/Classes/JsonConversions.h

```diff
@@ -5,15 +5,26 @@
 #import <Flutter/Flutter.h>
 #import <GoogleMaps/GoogleMaps.h>
 
-@interface FLTGoogleMapJsonConversions : NSObject
-+ (bool)toBool:(NSNumber *)data;
-+ (int)toInt:(NSNumber *)data;
-+ (double)toDouble:(NSNumber *)data;
-+ (float)toFloat:(NSNumber *)data;
-+ (CLLocationCoordinate2D)toLocation:(NSArray *)data;
-+ (CGPoint)toPoint:(NSArray *)data;
-+ (NSArray *)positionToJson:(CLLocationCoordinate2D)position;
-+ (UIColor *)toColor:(NSNumber *)data;
-+ (NSArray<CLLocation *> *)toPoints:(NSArray *)data;
-+ (NSArray<NSArray<CLLocation *> *> *)toHoles:(NSArray *)data;
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FLTGoogleMapJSONConversions : NSObject
```

Comment:
Nit: The file should be named `FLTGoogleMapJSONConversions.h`.

(Or even better, the name should stay the same and all of these should become C functions—since "classes" that are just class methods are an abomination IMO—at which point the name should be the same. But I know many ObjC developers like this style so I'm not going to die on that hill today.)